video2dataset/downloader.py

```diff
@@ -9,17 +9,19 @@
 
 def handle_youtube(youtube_url):
     """returns file and destination name from youtube url."""
-    ydl_opts = {}
+    yt_dlp.utils.std_headers['Referer'] = "https://www.youtube.com/"
+    yt_dlp.utils.std_headers['User-Agent'] = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36'
```

Comment:
what do these two lines do?

Reply:
I thought it would be better to send requests with referrer and headers so YouTube would not complain much but I can get rid of it if you wish